Tests/ExtractFilesTests.m

```diff
@@ -220,6 +220,23 @@ - (void)testExtractFiles_NoPasswordGiven
     }
 }
 
+- (void)testExtractFiles_IncorrectPassword
+{
+    NSString *testArchiveName = @"Test Archive (Bad Password).rar";
```

Comment:
Same as above, use the existing password-protected archives:

```
    NSArray *testArchives = @[@"Test Archive (Password).rar",
                              @"Test Archive (Header Password).rar"];
```

Reply:
Thank you for your comments. The reason I had to add a new archive is that for the existing ones it didn't return with 
bad password' error, rather then invalid header for some reason. I don't know how the existing ones have made. I created the 'new one' using rar command line tool and it gave me the expected error. Do you have any idea why does it work differently for the old ones?  Cheers

Reply:
Good call! It looks like this is a result of those being RAR < v5 archives. You can use "Test Archive (RAR5, Password).rar" and it works.